DRAFT.md

```diff
@@ -380,13 +380,13 @@ Copy constructors for an indirect value obtain an allocator by calling
 `allocator_traits<allocator_type>​::​select_on_container_copy_construction` on
 the allocator belonging to the indirect value being copied. Move constructors
 obtain an allocator by move construction from the allocator belonging to the
-container being moved. Such move construction of the allocator shall not exit
+container being moved. Such move construction of the allocator will not exit
```

Comment:
This bit is standardese and needs to be 'shall'

Reply:
I did wonder. :)